apigw-lambda-transcribe/README.md

```diff
@@ -0,0 +1,88 @@
+
+Once an audio file is uploaded, an S3 event invokes another Lambda function to start the Transcribe job using the StartTranscriptionJob API. Once the transcription is completed, the result will be stored in the output S3 bucket.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/apigw-lambda-transcribe
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Terraform](https://learn.hashicorp.cxom/tutorials/terraform/install-cli?in=terraform/aws-get-started) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd apigw-lambda-transcribe
+    ```
+1. From the command line, initialize terraform to downloads and installs the providers defined in the configuration:
+    ```
+    terraform init
+    ```
+1. From the command line, apply the configuration in the main.tf file:
+    ```
+    terraform apply
+    ```
+1. During the prompts
+    #var.prefix
+    - Enter a value: {enter any prefix to associate with resources}
+
+    #var.region
+    - Enter a value: {enter the region for deployment}
+
+## Testing
+
+1. Make a POST request to the API using the following cURL command:
+
+    curl --location 'https://<api-id>.execute-api.<region>.amazonaws.com/dev/generate-presigned-url' --header 'Content-Type: application/json' --data '{"object_name": "audio.mp3", "content_type": "audio/mpeg"}'
+
+    Note: Replace 'api-id' with the generated API ID from Terraform, 'region' with the region where the API is deployed (refer to the Terraform Outputs section) 'object_name' with your desired name for the S3 object and 'content_type' with the content type of the audio, for ex, mp3 or m4a
+
+1. Get the pre-signed URL from the previous step and use the following cURL command to upload the object in S3:
+
+    curl -v --location --request PUT '<presigned-url>' --header 'Content-Type: application/json' --data '<path-of-the-object>.mp3'
```

Comment:
The given command did not work for me, I had to use
```
curl -v --location -T "audio.mp3" \
  'PRESIGNED_URL' \
  --header 'Content-Type: audio/mpeg'
```
Can you please check?
I would recommend strings to be replaced in UPPERCASE.

Keeping the sample consistent in both code snippets (mp3, audio/mp3) is helpful to not confuse readers